src/SFA.DAS.Commitments.Application.UnitTests/Commands/UpdateApprenticeshipStopDate/WhenEditingTheStopDateOfAnApprenticeship.cs

```diff
@@ -203,6 +203,16 @@ public async Task ThenShouldPublishApprenticeshipEvent()
                     )), Times.Once);
         }
 
+        [Test]
+        public async Task ThenShouldPublishV2PublishApprenticeshipStopDateChangedEvent()
+        {
+            await Handler.Handle(ExampleValidRequest);
+
+            MockV2EventsPublisher.Verify(x =>
+                x.PublishApprenticeshipStopDateChanged(It.IsAny<Commitment>(), It.IsAny<Apprenticeship>()), Times.Once);
```

Comment:
It's worth verifying the call with the actual commitment and apprenticeship objects instead of `It.IsAny`.

Reply:
yes